src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -3413,6 +3413,35 @@ private double getDroppableTombstoneRatio(boolean useGcGrace)
         return allColumns > 0 ? allDroppable / allColumns : 0;
     }
 
+    public double getDroppableTombstoneCount()
+    {
+        return getDroppableTombstoneCount(true);
+    }
+
+    public double getTombstoneCount()
+    {
+        return getDroppableTombstoneCount(false);
+    }
+
+    public double getLiveTombstoneCount()
+    {
+        return getTombstoneCount() - getDroppableTombstoneCount();
+    }
+
+    private double getDroppableTombstoneCount(boolean useGcGrace)
```

Comment:
can you refactor `getDroppableTombstoneRatio` to use this instead for the numerator and only calculate the denominator itself ?

Reply:
actually i think it's more correct to calculate them together so they reference the same sstables instead of two separate calls to `getSSTables()`

Reply:
to avoid code duplication and unnecessary calculation, decided to use helper which takes in a view of sstables